### DISCUSSION TITLE
Item Name & Description not shown

### DISCUSSION THREAD

**Discussion by srtzero** [Digitalroot-Valheim/Atokal-AtosArrowsJVL#11] [Q&A] [answered]
### What happened?

The item name and item description do not show up in the workbench.  The only thing that shows up is the icons. In the workbench they names and description look like they are pulling from the actual plugin code. See screenshot.

### Screenshots

![image](https://user-images.githubusercontent.com/38842965/134512864-9806f2c9-c82c-480e-a4a3-b9ca7ab3af7f.png)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Digitalroot** [comment]
Are you using a mod manager?

**Comment by srtzero** [comment]
Yes. I am using Thunderstore R2modManager

**Comment by srtzero** [comment, marked as answer]
So seems to be an issue with the file on Thunderstore. I downloaded from NexusMods and everything is working as expected. Sorry  for the trouble. The one on the Thunderstore is ported by the OdinPlus Team.
